### ISSUE TITLE
Umenu dropdown is too small

### ISSUE THREAD

**Issue by 1offby** [Cycling74/miraweb#108]
The umenu dropdown is pretty tightly spaced. I can hit the correct selection but I sometimes miss.

<pre><code>
----------begin_max5_patcher----------
453.3oc0TssaCBCC8Y3q.kmYUDtzK6WYZpJPc6xFjTkKUsqp+6i3DZoRcapR
cOLj.isOX63yQbLNhTK2CZRxyIujDEcLNJBC4BDE7iHcr8MsLMBiX6.gkj5y
rioDrN3RhWnuNjaqBzfvvLbovAfFhyWgvk0u+TwYrLUeYLfZIHX0svX7ZyAe
.xsp7REzX7yed07IYoIzhJmohhNkSxRdM7cBamzZZACdRJFlGCzE1.zzDRJI
MIOXKB1xfsJXmFryB14magu9lCaAeEIbgoG.AuGOHbwvbPOuCLMuwEatmSzZ
ovn4ehMygIrwX6fUKYFihWaMvk2zAdMPrN9q0Bx0CgGhOlPZkhMeGGeEP8aR
k4Zj2B1vto3F43hcbMef+84OEGewfOOEm9Pzp2gPM+gJToyyPlbwBzjgd44+
mUpzhEiTp25H8GKJ+mnH63J1j0tpeGpO5uomx85oLjCxqlNIyc0y8z4eupZ3
uE7MBYesZ4MeLt+Ww7Y+.ymW555rY+buqYMerQIshUm2jgUGBgv1tcGnzgk.
t652buKUN2YonKW3cQhhn.G03wWhQXp9gyzOYVkmx2Osj3+T4JPIr7Pu667o
3u.66kJz
-----------end_max5_patcher-----------
</code></pre>

**Comment by fde31** [comment]
yep. We should improve that. Maybe we should even make it agnostic whether it's a touch enabled or mouse input enabled device in order to have better usability.

**Comment by starakaj** [comment]
Probably not too hard.
